app/models/user.rb

```diff
@@ -71,4 +75,26 @@ def generate_auth_token
       return token unless User.where(authentication_token: token).first
     end
   end
+
+  #
+  # Image helpers
+  #
+
+  def image_url
+    image_tmp_url || image.url
+  end
+
+  def image_tmp_url
+    "/tmp/uploads/#{image_tmp}" unless image_tmp.nil?
+  end
+
+  def self.convert_base64(data)
+    return unless data
+
+    temp_file = Tempfile.new [Devise.friendly_token, "jpg"]
+    temp_file.binmode
+    temp_file.write(Base64.decode64(data))
+
+    ActionDispatch::Http::UploadedFile.new(tempfile: temp_file, filename: "#{Devise.friendly_token}.jpg")
+  end
```

Comment:
Is this necessary to put in the `User` class? This looks like a general method that doesn't depend on anything specific to `User`. If so, we can move this into a concern.